README.md

```diff
@@ -5,11 +5,12 @@ Attempt to bundle the Desmos API using Webpack
 ## Instructions
 
 1. Install [NodeJS v8.X (LTS)](https://nodejs.org/en/download/)
-2. Copy a version of `calculator.js` into the `src/` folder
-2. At command prompt, run:  
-```cmd
-npm install
-npm run
-```
-3. Browser should open at http://localhost:8085
-4. Check the console for errors
\ No newline at end of file
+1. Copy a version of `calculator.js` into the `vendor/` folder
+1. Generate a copy of calculator.js with references to window.Desmos removed
+and place it in the src directory using `sed 's/window.Desmos/Desmos/g' < vendor/calculator.js > src/calculator.js`
```

Comment:
Note the new instructions here. This is a temporary workaround--I'm going to do some more work over here to deliver something better.